test/image.test.js

```diff
@@ -106,6 +106,14 @@ describe('Image', function () {
     })
   })
 
+  it('returns raw image data', function() {
+      var img = new Image();
+      img.src = png_clock;
+      var buf = img.rawData;
+      assert.ok(buf instanceof Buffer);
+      assert.strictEqual(409600, img.rawData.length);
+  });
```

Comment:
Please follow standard style here (2 spaces indentation, space between `function` and `()`, no semicolons)